File: MeetupBooking.WebApi/Models/Meetup/MeetupViewModel.cs
using MeetupBooking.WebApi.Models.Room;
using System.Collections.Generic;

namespace MeetupBooking.WebApi.Models.Meetup
{
    public class MeetupViewModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string OwnerName { get; set; }

        public ICollection<RoomViewModel> Rooms { get; set; }

        public ICollection<ParticipantViewModel> Participants { get; set; }
    }
}

File: MeetupBookingWeb/src/app/core/models/meetupViewModel.ts
export class MeetupViewModel  {

    id: number;
    name: string;
    description: string;
    ownerName: string;
    rooms: Room[];
    participants: Participant[];
}

export class Participant{
    id: number;
    name: string;
}

export class Room{
    id: number;
    name: string;
}